crates/viewer/re_viewer/src/open_url.rs

```diff
@@ -49,6 +58,23 @@ impl std::str::FromStr for ViewerImportUrl {
     /// * intra-recording links (typically links to an entity)
     /// * web event listeners
     fn from_str(url: &str) -> Result<Self, Self::Err> {
+        // This might be a web-viewer URL with `url` parameters.
+        // Extract the `url` parameter and call this function again.
+        if let Ok(url) = url::Url::parse(url) {
+            // It's rare, but there might be *several* `url` parameters.
+            let url_params = url
+                .query_pairs()
+                .filter_map(|(key, value)| (key == "url").then(|| Self::from_str(&value)))
+                .collect::<anyhow::Result<Vec<_>>>()?;
+
+            if let Ok(url_parameters) = vec1::Vec1::try_from_vec(url_params) {
+                return Ok(Self::WebViewerUrl {
+                    base_url: url.origin(),
+                    url_parameters,
+                });
+            }
+        }
```

Comment:
this being rather general ("anything with `url` GET param"), it should go after the more specific stuff like RedapEntry etc.